dDatabase/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ add_subdirectory(CDClientDatabase)
 add_subdirectory(GameDatabase)
 
 add_library(dDatabase STATIC "MigrationRunner.cpp")
+
+add_custom_target(conncpp_dylib
+	${CMAKE_COMMAND} -E copy $<TARGET_FILE:MariaDB::ConnCpp> ${PROJECT_BINARY_DIR})
```

Comment:
This is a bit of kludge. It's being placed in the correct directory for a multi-config generator. The problem is our scripts aren't set up to handle that correctly. We shouldn't need this.

Reply:
It is not being placed in the correct directory for milti-config generators as for mac CI and development, the file is requires to be in the linking directory and the binary directory at these times, and in neither was this true before.